controllers/shipwrightbuild_controller.go

```diff
@@ -131,13 +149,29 @@ func (r *ShipwrightBuildReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	logger.Info("Applying manifest's resources...")
 	if err := manifest.Apply(); err != nil {
 		logger.Error(err, "Rolling out manifest's resources")
+		apimeta.SetStatusCondition(&b.Status.Conditions, metav1.Condition{
+			Type:    ConditionReady,
+			Status:  metav1.ConditionFalse,
+			Reason:  "Failed",
+			Message: "Reconciling ShipwrightBuild failed",
+		})
+		r.Client.Status().Update(ctx, b)
 		return RequeueWithError(err)
 	}
 	if err := r.setFinalizer(ctx, b); err != nil {
 		logger.Info(fmt.Sprintf("%#v", b))
 		logger.Error(err, "Setting the finalizer")
 		return RequeueWithError(err)
 	}
+	apimeta.SetStatusCondition(&b.Status.Conditions, metav1.Condition{
+		Type:    ConditionReady,
+		Status:  metav1.ConditionTrue,
+		Reason:  "Success",
+		Message: "Reconciling ShipwrightBuild successfully",
```

Comment:
Use past tense here:
```suggestion
		Message: "Reconciled ShipwrightBuild successfully",
```

Reply:
done